modules/telco-troubleshooting-general-check-logs.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-general-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-troubleshooting-general-check-logs_{context}"]
+= Checking pod logs
+
+Get logs from the pod.
+
+.Procedure
+
+. List the pods by running the following command:
++
+--
+[source,terminal]
+----
+$ oc get po
```

Comment:
Should the command be `$ oc get pod`?

Reply:
It's a shorter version of `oc get pod`. Even though it's valid, I updated the instances to `oc get pod` to avoid any confusion